bookkeeper-http/servlet-http-server/src/main/java/org/apache/bookkeeper/http/servlet/JettyHttpServer.java

```diff
@@ -0,0 +1,59 @@
+package org.apache.bookkeeper.http.servlet;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.nio.file.Files;
+import java.util.List;
+import javax.servlet.Servlet;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.handler.ContextHandlerCollection;
+import org.eclipse.jetty.servlet.ServletHolder;
+import org.eclipse.jetty.webapp.WebAppContext;
+
+/**
+ * Jetty based http server
+ **/
+
+public class JettyHttpServer {
```

Comment:
Please move this class to the tests sources.
This is not intended to be used

Reply:
Ok!

Reply:
> Please move this class to the tests sources.
> This is not intended to be used

done